go.mod

```diff
@@ -4,12 +4,12 @@ go 1.23.1
 
 require (
 	github.com/avast/retry-go v2.7.0+incompatible
-	github.com/containerd/containerd/v2 v2.0.1
+	github.com/containerd/containerd v1.7.23
```

Comment:
why is this downgraded?

Reply:
renovate is stupid with this versions..... it keep updating it like it did vfs/v4 and vfs/v5 but the old version still in use. For big bumps liek this we need to manually bump them

Reply:
I think for `go mod`, `github.com/containerd/containerd/v2` is a different package from `github.com/containerd/containerd`, os what actually happens here is, that v2 was not required anywhere, thus it's removed and `go mod tidy` brings back `containerd` (without `v2`) which is required.

I could be wrong, but that's what I think is happening. `renovate` shouldn't had bumped this in the first place.